azure-pipelines.yml

```diff
@@ -40,6 +40,30 @@ jobs:
         COMPILER: 'clang++'
         TEST_TARGET: 'linux_clang10'
         HOST_CONFIG: 'clang@10.0.0'
+      linux_clang10_noraja:
+        VM_ImageName: 'ubuntu-20.04'
+        Compiler_ImageName: '$(CLANG10_IMAGENAME)'
+        CMAKE_EXTRA_FLAGS: '-DBUILD_SHARED_LIBS=ON -DAXOM_QUEST_ENABLE_EXTRA_REGRESSION_TESTS:BOOL=ON -U RAJA_DIR'
+        BUILD_TYPE: 'Debug'
+        COMPILER: 'clang++'
+        TEST_TARGET: 'linux_clang10'
+        HOST_CONFIG: 'clang@10.0.0'
+      linux_clang10_noumpire:
+        VM_ImageName: 'ubuntu-20.04'
+        Compiler_ImageName: '$(CLANG10_IMAGENAME)'
+        CMAKE_EXTRA_FLAGS: '-DBUILD_SHARED_LIBS=ON -DAXOM_QUEST_ENABLE_EXTRA_REGRESSION_TESTS:BOOL=ON -U UMPIRE_DIR'
+        BUILD_TYPE: 'Debug'
+        COMPILER: 'clang++'
+        TEST_TARGET: 'linux_clang10'
+        HOST_CONFIG: 'clang@10.0.0'
+      linux_clang10_noraja_noumpire:
+        VM_ImageName: 'ubuntu-20.04'
+        Compiler_ImageName: '$(CLANG10_IMAGENAME)'
+        CMAKE_EXTRA_FLAGS: '-DBUILD_SHARED_LIBS=ON -DAXOM_QUEST_ENABLE_EXTRA_REGRESSION_TESTS:BOOL=ON -U RAJA_DIR -U UMPIRE_DIR'
+        BUILD_TYPE: 'Debug'
+        COMPILER: 'clang++'
+        TEST_TARGET: 'linux_clang10'
+        HOST_CONFIG: 'clang@10.0.0'
```

Comment:
These three new checks are not showing up as Required.  Any ideas?  The whole reason we have them is to make sure they succeed!

Reply:
I see them here: https://dev.azure.com/healy22/Status%20Check%20Testiing/_build/results?buildId=7847&view=results

![image](https://user-images.githubusercontent.com/5626552/236066726-381995ae-1cc9-4e3b-ab51-9c72beaad25f.png)

Reply:
The Branch Protection rules controls this. The parent job "LLNL.axom" covers those jobs but I can add them explicitly like the rest of the jobs after it is merged and ran on the default branch ("develop").

Reply:
Thanks, Chris.  Yes, please, after it gets merged in.

Reply:
(Sorry -- I misread the "Required" part)